.gitignore

```diff
@@ -6,6 +6,7 @@
 
 *.log
 *_ui.py
+.vscode/*
```

Comment:
I should have been more specific. Look at how the `grid-tied` repository implements. See the files `.gitignore` and `.gitignore.local`. Add this section:

```
#Visual Studio and Visual Studio Code
#The templates are not currently adequate.
.vs/
.vscode/
```

Reply:
[1107208](https://github.com/epcpower/m-pm/pull/3/commits/1107208290999e76efef7d7c2757a4b40799ffb8)